src/Microsoft.DotNet.Interactive.CSharpProject/RoslynWorkspaceUtilities/RoslynWorkspaceUtilities.cs

```diff
@@ -47,7 +47,7 @@ internal static BuildDataResults GetResultsFromCacheFile(string cacheFilePath)
                 CSharpParseOptions = cSharpParseOptions
             };
         }
-        catch (Exception)
+        catch (ArgumentNullException)
         {
             return null;
```

Comment:
Why are we hiding an exception here at all? This is presumably happening due to a bug in our code.